### ISSUE TITLE
need updated top level readme for blue pearl app

### ISSUE THREAD

**Issue by GregDaughtry** [Xilinx/XilinxTclStore#597]
Hi there.  Your readme.md in the top level of the blue pearl app appears to be the "My Company" template.  Can you please update it with a Blue pearl specific description of your app?

**Comment by scottaronbloom** [comment]
Yes, btw, there are multiple apps that have the same template..

**Comment by dpefour** [comment]
That’s right. It something that we should have tracked better in the past.

Thank you, David
